data/blog/search.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Search
+author: 
+  - echjordan
+  - vanessayuenn
+  - zeke
+date: '2018-06-21'
+---
+
+The Electron website has a new search engine that delivers instant results for
+API docs, tutorials, Electron-related npm packages, and more.
+
+<figure>
+  <a href="https://electronjs.org/?query=resize" style="display: block; text-align: center;">
+    <img class="screenshot" src="https://user-images.githubusercontent.com/2289/41683719-417ca80a-7490-11e8-9a52-fb145f4251ba.png" alt="Electron Search Screenshot">
+  </a>
+</figure>
+
+---
+
+Learning a new technology or framework like Electron can be intimidating. 
+Once you get past the [quick-start] phase, it can 
+be difficult to learn best practices, find the right APIs, or discover the tools 
+that will help you build the app of your dreams. As app developers, we often find 
+ourselves turning to sites like  Google and Stack Overflow for answers. This 
+works, but we can do better.
```

Comment:
I don't like this framing: Stack Overflow is a complementary tech, so it's not a competitor and we're not trying to 'do better' than them.

Reply:
Good call.